releasenotes/notes/sqisw-decomposition-2478b9f0b7c54044.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Added a new gate to the standard_gates library :class:`.SiSwapGate`
+    to represent the square-root of an iSWAP. This gate has the desirable
+    properties of being faster/higher-fidelity than a full iSWAP, but also
+    synthesizing the volume of two-qubit unitaries with fewer applications.
+
+    An associated decomposition method that synthesizes arbitrary two-qubit
+    unitaries into two or three applications of SiSwap has been added in
+    :class:`.SiSwapDecomposer`.
```

Comment:
It would be cool to show an example in the release notes, too 😄

Reply:
I'll add.